### ISSUE TITLE
Failed training plot for a linear model

### ISSUE THREAD

**Issue by Bleyddyn** [tawnkramer/donkey#16]
A recent change to train.py added a plot for steering accuracy, but that fails on linear models that can't calculate accuracy.

I have a quick, hacky fix that can be copied over or I can turn it into a PR: https://github.com/Bleyddyn/donkey/commit/33f7da604d76e461b1ca545dff58368aa8e6994f.

Probably a better long term solution is to have each KerasPilot sub-class return a list of dictionaries with parameters for each relevant plot: title, axis labels, legend label(s), history key(s). Adding a plot method to KerasPilot would also work, but then keras.py would need to become dependent on matplotlib.

**Comment by tawnkramer** [comment]
Thanks for the heads up. Your changes look good for now. If you want to a pull request, that would be helpful.

**Comment by Bleyddyn** [comment]
Haven't heard any complaints, so I'm closing this. Fixed in https://github.com/tawnkramer/donkey/pull/17
